.github/workflows/pull_request.yml

```
@@ -22,3 +22,33 @@ jobs:
     needs: call_build
     uses: ./.github/workflows/cypress_test.yml
     secrets: inherit
+
+  deploy:
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event.pull_request.head.repo.full_name == github.repository }}  
+    needs: [call_build, ember_test, cypress_test]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    environment: vercel-bracco-preview
+    env:
```

actionlint (1.7.11)

[error] expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)